README.md

```diff
@@ -243,6 +249,72 @@ foreach ($scoreResponse->warnings as $warning) {
 }
 ```
 
+### Report Transactions API ###
+
+MaxMind encourages the use of this API as data received through this channel is
+used to continually improve the accuracy of our fraud detection algorithms.
+
+To use the Report Transactions API, create a new `\MaxMind\ReportTransaction`
+object. The constructor takes your MaxMind account ID, license key, and an
+optional options array as arguments. This object is immutable. You then send one
+or more reports using the `->report` method as shown below.
+
+If there is a validation error in the data passed to the `->report` method, a
+`\MaxMind\Exception` will be thrown. This validation can be disabled by
+setting `validateInput` to `false` in the options array for
+`\MaxMind\ReportTransaction`, but it is recommended that you keep it on at least
+through development as it will help ensure that you are sending valid data to
+the web service.
+
+If the report is successful, nothing is returned. If the report fails, an
+exception with be thrown.
+
+See the API documentation for more details.
+
+#### Report Transaction Exceptions ####
+
+All externally visible exceptions are in the `\MaxMind\Exception` namespace.
+The possible exceptions are:
+
+* `InvalidInputException` - This will be thrown when the `->report()` method is
+  called with invalid input data or when the required `ip_address` or `tag`
+  fields are missing.
+* `AuthenticationException` - This will be thrown on calling `->report()`,
+  when the server is unable to authenticate the request, e.g., if the license
+  key or account ID is invalid.
+* `InvalidRequestException` - This will be thrown on calling `->report()` when
+  the server rejects the request for another reason such as invalid JSON in the
+  POST.
+* `HttpException` - This will be thrown on calling `->report()` when an
+  unexpected HTTP error occurs such as a firewall interfering with the request
+  to the server.
+* `WebServiceException` - This will be thrown on calling `->report()` when some
+  other error occurs. This also serves as the base class for the above
+  exceptions.
+
+#### Report Transaction Example ####
+
+```php
+<?php
+require_once 'vendor/autoload.php';
+use MaxMind\ReportTransaction;
+
+# The constructor for ReportTransaction takes your account ID, your license key,
+# and optionally an array of options.
+$rt = new ReportTransaction(1, 'ABCD567890');
+
+$rt->report(
+	'ip_address'      => '81.2.69.160',
+    'tag'             => 'chargeback',
```

Comment:
The formatting on this looks off in GitHub. Maybe you are switching between tabs and spaces?